client/app/lib/components/Pagination.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useEffect } from 'react';
+import { Pagination } from '@mui/material';
+
+interface Props {
+  items: any[];
```

Comment:
Can we type this out?

Reply:
Not sure what you mean by this. Should I make a new type in the `types` folder for this?